### PR TITLE
Add workflow to label PR's based on size

### LINKED ISSUE #1404: Create GitHub Action to label pull requests based on number of lines changed
I happened across a [GitHub Action that will label pull requests based on the number of lines changed](https://github.com/marketplace/actions/add-size-label-to-pr), and I'm thinking that this would be a good one to add to our automated tasks. 

This GitHub Action would be particularly helpful in helping us triage pull requests for code reviews.  For example, if we're hypothetically in the middle of a global pandemic and are dealing with huge amounts of brain blah, we could pick shorter pull requests to review and get in quickly, and then know to set aside a block of time and coffee for reviewing in-depth pull requests.

This would quite possibly be a good first pull request for someone who would like to gain some experience about GitHub Actions.

### PR DESCRIPTION
**Pull request by RAJAGOPALAN-GANGADHARAN**
Using https://github.com/marketplace/actions/add-size-label-to-pr to add size labels to PR's. Triggers when new PR's are created and the existing ones are updated.

- [ ] I have added a changelog entry for this pull request.

Closes #1404

### CODE REVIEW

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
pre-commit.ci autofix

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -6,7 +6,13 @@ jobs:
...
       with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
-        sync-labels: true
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          sync-labels: true
+    - name: Add Size Label to PR
+      uses: WizardOhio24/label-pr-size-action@master
+      with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          size-label-color: '10, very small, ffaf65 | 50, small, ff9e58 | 80, medium, ff9654 | 200, large, ff905d | 400, massive, ff905d'
```
Comment (namurphy) [inline .github/workflows/labeler.yml:18]:
```suggestion
          size-label-color: '10, very small, ffaf65 | 50, small, ff9e58 | 200, medium, ff9654 | 400, large, ff905d | 100000, massive, ff905d'
```
What would you think of this adjustment to the numbers?  I'm remembering a [study](https://smartbear.com/learn/code-review/best-practices-for-peer-code-review/) that found that code reviews should ideally cover ≲200–400 lines of code.  This change would make "large" PRs in the range of 200–400, and increase the upper limit for massive PRs.

**Review by namurphy — COMMENTED** [review]
Thank you for doing this!  I have two quick minor suggestions, and was thinking that it'd be helpful to rename `1467.feature.rst` to `1467.trivial.rst` since this change is not going to affect end users.  Thank you again!

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
@namurphy Thank you for reviewing this PR, you are right somehow I missed the trivial changelog category. Will update the ticket ranges as well.

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
pre-commit.ci autofix

**Review by namurphy — APPROVED** [review]

**Comment by namurphy** [comment]
Looks good!  Let's try it out.  Thank you for doing this!

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
https://github.com/PlasmaPy/PlasmaPy/runs/5612561504?check_suite_focus=true -> Just checked with a test PR it's not worrking, will try to debug it. Also one other point pre-commit doesn't autofix pretty-format-yaml failures. Perhaps we can try to add this as well to pre commit.

**Pull request merged**